keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -563,6 +579,34 @@ For example, if I maintain `example.com/my-signer`, I can use the following stra
   canary object first, and assess the health of the canary workloads.
 * Once I am satisfied that the change is safe, I edit the live object.
 
+### Publishing the kube-apiserver Trust Bundle
+
+Today, the trust bundle that allows verifying kube-apiserver serving certificate(s)
+at its internal endpoints is distributed into every namespace using a configMap.
+This is so that it can be mounted along with the ServiceAccount token in order
+for the workloads to be able to communicate with the kube-apiserver.
+
+To avoid the need for these configMaps in the future, a ClusterTrustBundle API
+object will be created for all clusters:
+
+```yaml
+apiVersion: v1beta1
+kind: ClusterTrustBundle
+metadata:
+  name: kubernetes.io:kube-apiserver-server:live # TODO: should we use "active" instead of "live", maybe?
```

Comment:
I'm more inclined to not pin the suffix at all and make consumers rely on the signerName selector